src/_zkapauthorizer/validators.py

```diff
@@ -22,10 +22,10 @@
 def is_base64_encoded(b64decode=b64decode):
     def validate_is_base64_encoded(inst, attr, value):
         try:
-            b64decode(value.encode("ascii"))
+            b64decode(value)
```

Comment:
Do you mind adding a docstring and/or type comment to `is_base64_encoded` that reflects this change?